### ISSUE TITLE
[BUG]: Server Crash after intro finish

### ISSUE THREAD

**Issue by luislclt** [norimicry/stardew-multiplayer-docker#40]
**Describe the bug**
Server was configured via VNC, but crash after starting multiplayer mode message, then got VNC black screen. 
On the logs, this append right after logs message: "INFO Always On Server] Auto Mode On!"

Edit env info:
Linux VM running Ubuntu Server 22
Use of SteamCMD login, disabled MFA

**To Reproduce**
1. Clean docker enviroment, then  `docker compose -f docker-compose-steam.yml up`
2. Proceed to VNC page & configure Host options 
![image](https://github.com/user-attachments/assets/6875c950-9821-43c8-afa4-e2ece14535f1)

3. Press OK then wait for the intro to finish,  then VNC get black screen! 
![image](https://github.com/user-attachments/assets/114f9edb-4e67-4f9a-af35-a03c4c0dee28)

4. Lastest server logs before crash:
![image](https://github.com/user-attachments/assets/ba76b107-ff77-4c9d-91e4-3f335db89f85)

**Logs**
Empty spaces are some logs from VNC...

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.\n
. . . 
31/07/2024 19:57:53 ------------------ USEFUL INFORMATION ------------------\n
[app] starting StardewValley...\n
. . . 
31/07/2024 19:57:53 Default visual ID: 0x21\n
Configuring /data/Stardew/game/Mods/Always On Server/config.json\n
Configuring /data/Stardew/game/Mods/AutoLoadGame/config.json\n

Removing /data/Stardew/game/Mods/ChatCommands/ (ENABLE_CHATCOMMANDS_MOD=false)\n

Removing /data/Stardew/game/Mods/ConsoleCommands/ (ENABLE_CONSOLECOMMANDS_MOD=false)\n

Removing /data/Stardew/game/Mods/Crops Anytime Anywhere/ (ENABLE_CROPSANYTIMEANYWHERE_MOD=false)\n
Removing /data/Stardew/game/Mods/FriendsForever/ (ENABLE_FRIENDSFOREVER_MOD=false)\n
Removing /data/Stardew/game/Mods/NonDestructiveNPCs/ (ENABLE_NONDESTRUCTIVENPCS_MOD=false)\n
Removing /data/Stardew/game/Mods/SaveBackup/ (ENABLE_SAVEBACKUP_MOD=)\n
Removing /data/Stardew/game/Mods/TimeSpeed/ (ENABLE_TIMESPEED_MOD=false)\n
Configuring /data/Stardew/game/Mods/UnlimitedPlayers/config.json\n
-- SMAPI Log: Starting\n
-- SMAPI Log: Waiting for log to appear\n
Running Stardew Valley\n
chmod: changing permissions of 'start.sh': Operation not permitted\n
/usr/bin/xterm\n
-- SMAPI Log:  Tailing\n
[19:57:56 INFO  SMAPI] SMAPI 4.0.8 with Stardew Valley 1.6.8 build 24119 on Unix 6.8.0.39\r\n
[19:57:56 INFO  SMAPI] Mods go here: /data/Stardew/game/Mods\r\n
[19:57:56 TRACE SMAPI] Log started at 2024-07-31T19:57:56 UTC\r\n
[19:58:04 DEBUG SMAPI] Waiting for game to launch...\r\n
[19:58:04 ERROR game] Game.Initialize() caught exception initializing XACT.\n
Microsoft.Xna.Framework.Audio.NoAudioHardwareException (0x80004005)\n
   at Microsoft.Xna.Framework.Audio.SoundEffect.Device() in D:\\GitlabRunner\\builds\\Gq5qA5P4\\1\\ConcernedApe\\stardewvalley\\MonoGame.Desktop\\MonoGame.Framework\\Platform\\Audio\\SoundEffect.FAudio.cs:line 259\n
   at Microsoft.Xna.Framework.Audio.SoundEffect.Initialize() in D:\\GitlabRunner\\builds\\Gq5qA5P4\\1\\ConcernedApe\\stardewvalley\\MonoGame.Desktop\\MonoGame.Framework\\Audio\\SoundEffect.cs:line 191\n
   at Microsoft.Xna.Framework.Audio.SoundEffect..ctor(MiniFormatTag codec, Byte[] buffer, Int32 channels, Int32 sampleRate, Int32 blockAlignment, Int32 loopStart, Int32 loopLength) in D:\\GitlabRunner\\builds\\Gq5qA5P4\\1\\ConcernedApe\\stardewvalley\\MonoGame.Desktop\\MonoGame.Framework\\Audio\\SoundEffect.cs:line 148\n
   at Microsoft.Xna.Framework.Audio.WaveBank..ctor(AudioEngine audioEngine, String waveBankFilename, Boolean streaming, Int32 offset, Int32 packetsize) in D:\\GitlabRunner\\builds\\Gq5qA5P4\\1\\ConcernedApe\\stardewvalley\\MonoGame.Desktop\\MonoGame.Framework\\Audio\\Xact\\WaveBank.cs:line 276\n
   at StardewValley.Game1.Initialize() in D:\\GitlabRunner\\builds\\Gq5qA5P4\\1\\ConcernedApe\\stardewvalley\\Farmer\\Farmer\\Game1.cs:line 2191\r\n
[19:58:04 DEBUG SMAPI] Loading mod metadata...\r\n
[19:58:05 DEBUG SMAPI] Loading mods...\r\n
[19:58:05 TRACE SMAPI]    Auto Load Game (from Mods/AutoLoadGame/AutoLoadGame.dll, ID: caraxian.AutoLoadGame)...\r\n
[19:58:05 TRACE SMAPI]       Failed loading PDB for 'AutoLoadGame.dll'. Technical details:\n
Mono.Cecil.Cil.SymbolsNotMatchingException: Symbols were found but are not matching the assembly\n
   at Mono.Cecil.ModuleDefinition.ReadSymbols(ISymbolReader reader, Boolean throwIfSymbolsAreNotMaching)\n
   at Mono.Cecil.ModuleReader.ReadSymbols(ModuleDefinition module, ReaderParameters parameters)\n
   at Mono.Cecil.ModuleReader.CreateModule(Image image, ReaderParameters parameters)\n
   at StardewModdingAPI.Framework.ModLoading.AssemblyLoader.GetReferencedLocalAssemblies(FileInfo file, HashSet`1 visitedAssemblyNames, IAssemblyResolver assemblyResolver)+MoveNext() in E:\\source\\_Stardew\\SMAPI\\src\\SMAPI\\Framework\\ModLoading\\AssemblyLoader.cs:line 281\r\n
[19:58:05 TRACE SMAPI]    Always On Server (from Mods/Always On Server/Always On Server.dll, ID: mikko.Always_On_Server)...\r\n
[19:58:05 TRACE SMAPI]       Detected reference to StardewModdingAPI.Events.ISpecializedEvents.UnvalidatedUpdateTicked event in assembly Always On Server.dll.\r\n
[19:58:05 TRACE SMAPI]       Detected direct console access (System.Console type) in assembly Always On Server.dll.\r\n
[19:58:05 TRACE SMAPI]    Unlimited Players by Armitxes (from Mods/UnlimitedPlayers/UnlimitedPlayers.dll, ID: Armitxes.UnlimitedPlayers)...\r\n
[19:58:05 TRACE SMAPI]    No Fence Decay (from Mods/NoFenceDecay/NoFenceDecay.dll, ID: cat.nofencedecay)...\r\n
[19:58:05 TRACE SMAPI]       Rewrote NoFenceDecay.dll for OS...\r\n
[19:58:05 TRACE SMAPI]       Rewrote NoFenceDecay.dll to fix BuildableGameLocation.buildings field, OverlaidDictionary.get_Values method, StardewValley.Network.OverlaidDictionary/ValuesCollection type...\r\n
[19:58:05 TRACE SMAPI]       Rewrote NoFenceDecay.dll to fix Game1.locations (field =\u003e property)...\r\n
[19:58:05 TRACE SMAPI]       Rewrote NoFenceDecay.dll to fix 32-bit architecture...\r\n
[19:58:06 INFO  SMAPI] Loaded 4 mods:\r\n
[19:58:06 INFO  SMAPI]    Always On Server 1.20.3-unofficial.2-mikkoperkele by funny-snek \u0026 Zuberii | A Headless server mod.\r\n
[19:58:06 INFO  SMAPI]    Auto Load Game 1.0.3 by Caraxian, yatsukiko | Automatically load a save file when starting.\r\n
[19:58:06 INFO  SMAPI]    No Fence Decay 1.5.0 by Cat | Makes all fences and gates last forever.\r\n
[19:58:06 INFO  SMAPI]    Unlimited Players by Armitxes 2024.4.16 by Armitxes | Remove the maximum player restrictions, build endless cabins.\r\n
\r\n
[19:58:06 INFO  SMAPI] Found 1 mod with warnings:\r\n
[19:58:06 INFO  SMAPI]    Bypassed safety checks\r\n
[19:58:06 INFO  SMAPI]    --------------------------------------------------\r\n
[19:58:06 INFO  SMAPI]       These mods bypass SMAPI's normal safety checks, so they're more likely to cause errors or save\r\n
[19:58:06 INFO  SMAPI]       corruption. If your game has issues, try removing these first.\r\n
\r\n
[19:58:06 INFO  SMAPI]       - Always On Server\r\n
\r\n
[19:58:06 TRACE SMAPI]    Direct console access\r\n
[19:58:06 TRACE SMAPI]    --------------------------------------------------\r\n
[19:58:06 TRACE SMAPI]       These mods access the SMAPI console window directly. This is more fragile, and their output may not\r\n
[19:58:06 TRACE SMAPI]       be logged by SMAPI.\r\n
\r\n
[19:58:06 TRACE SMAPI]       - Always On Server\r\n
\r\n
[19:58:06 DEBUG SMAPI] Launching mods...\r\n
[19:58:06 INFO  Unlimited Players by Armitxes] Default player limit set to 10 players.\r\n
[19:58:06 DEBUG SMAPI] Mods loaded and ready!\r\n
[19:58:06 TRACE SMAPI] Checking for updates...\r\n
[19:58:07 TRACE game] setGameMode( 'titleScreenGameMode (0)' )\r\n
[19:58:07 TRACE game] path '/config/xdg/config/StardewValley/startup_preferences' did not exist and will be created\r\n
[19:58:07 TRACE SMAPI]    SMAPI okay.\r\n
[19:58:07 TRACE SMAPI]    Checking for updates to 4 mods...\r\n
[19:58:07 ERROR game] Oops! Steam achievements won't work because Steam isn't loaded. You can launch the game through Steam to fix that.\r\n
[19:58:07 INFO  SMAPI] Type 'help' for help, or 'help \u003ccmd\u003e' for a command's usage\r\n
[19:58:07 TRACE SMAPI] Got update-check errors for some mods:\n
   Unlimited Players by Armitxes: The CurseForge mod with ID '362847' has no valid versions.\r\n
[19:58:07 TRACE SMAPI]    All mods up to date.\r\n
[19:58:07 INFO  Unlimited Players by Armitxes] \n
[SERVER] Adjusting limit to 10 players.\r\n
[19:58:07 INFO  Auto Load Game] Loading Save: VinhoVerdePT_382124946\r\n
[19:58:07 TRACE game] gameMode was 'titleScreenGameMode (0)', set to 'loadingMode (6)'.\r\n
[19:58:07 TRACE SMAPI] Game loader synchronizing...\r\n
[19:58:07 TRACE game] getLoadEnumerator('VinhoVerdePT_382124946')\r\n
[19:58:08 TRACE game] regular stream detected...\r\n
[19:58:08 TRACE SMAPI] Synchronizing 'Load_Deserialize' task...\r\n
[19:58:14 TRACE SMAPI]    task complete.\r\n
[19:58:14 TRACE SMAPI] Synchronizing 'Load_LoadForNewGame' task...\r\n
[19:58:16 TRACE SMAPI]    task complete.\r\n
[19:58:16 TRACE SMAPI] Synchronizing 'Load_Farmer' task...\r\n
[19:58:16 TRACE SMAPI]    task complete.\r\n
[19:58:16 TRACE SMAPI] Synchronizing 'Load_Locations' task...\r\n
[19:58:17 TRACE SMAPI]    task complete.\r\n
[19:58:17 TRACE game] Starting server. Protocol version: 1.6.6\r\n
[19:58:17 TRACE game] Starting LAN server\r\n
[19:58:17 TRACE game] gameMode was 'loadingMode (6)', set to 'playingGameMode (3)'.\r\n
[19:58:17 TRACE game] getLoadEnumerator() exited, elapsed = '00:00:09.5548961'\r\n
[19:58:17 TRACE SMAPI] Game loader done.\r\n
[19:58:18 TRACE SMAPI] Context: loaded save 'VinhoVerdePT_382124946', starting spring 1 Y1, locale set to . Main player with 1 player online.\r\n
[19:58:18 INFO  Always On Server] Auto Mode On!\r\n
31/07/2024 19:59:47 Got connection from client 127.0.0.1\n
31/07/2024 19:59:47   other clients:\n

**Additional context**
Let me know people if more of you have the same issue, I'm happy to provide more feedback and to do trobleshooting.

**Comment by cavazos-apps** [comment]
What kind of system are you hosting the container on?

**Comment by luislclt** [comment]
Proxmox VM running Ubuntu 24.04 LTS
Docker Compose version v2.29.1
Docker Engine - Community
 Version:           27.1.1
 API version:       1.46
 Go version:        go1.21.12
 Git commit:        6312585
 Built:             Tue Jul 23 19:57:14 2024
 OS/Arch:           linux/amd64
 Context:           default

Let me know if you need more details

**Comment by luislclt** [comment]
Good news, I did clean up the folder project & clean docker resources, and do a git clone again, and start with all default settings. It works again.

used this commands for the clean up did the trick:

docker stop $(docker ps -qa)
docker rm $(docker ps -qa)
docker rmi $(docker images -qa)
docker builder prune
docker system prune -a -f

Working print:
![image](https://github.com/user-attachments/assets/2064c52d-c662-4601-b5ff-0dfcbe6e3f18)

Will keep loking what misconfiguration or changes I have made previously, if problems appear I will let you know again.
